### PR TITLE
12.0 add res company category

### PR DESCRIPTION
**Pull request by legalsylvain**
Add trivial module ``res_company_category`` that add a new model that allow to set category to companies. This is a module usefull in a multi company context, to analyse datas.

res_company_category_tree

res_company_form

### CODE REVIEW

File: res_company_category/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#   * res_company_category
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
```
Comment (fcayre) [inline res_company_category/i18n/fr.po]: 8.0 => 12.0?

**Review by fcayre — APPROVED** [review]
LGTM (not tested functionally)

File: res_company_category/i18n/fr.po (hunk 1 above)
Comment (legalsylvain) [reply res_company_category/i18n/fr.po]: thanks. translation updated.

**Review by bealdav — APPROVED** [review]
Just tested: it works
light code review but I know the author

Thanks

**Pull request merged**